bin/resources/GameIndex.yaml

```diff
@@ -32209,6 +32212,9 @@ SLPS-20131:
 SLPS-20132:
   name: "Chenuen no San Goku Shi (Chen Wen's Romance of the Three Kingdoms)"
   region: "NTSC-J"
+SLPS-20135:
+  name: "Pai Chanjong"
```

Comment:
The interwebs seems set on it being Chenjan rather than Chanjong

Reply:
I went with the name redump.org used.
http://redump.org/disc/77971/
チェンジャン is read as chenjan, but the ジャン is mostly written in romaji as jong when it is related to Mahjong (which is Mājan in Japanese). So, personally I would transliterate the title as chenjong (I have no idea why redump went with chan instead of chen).
Pai Chenjan or Pai Chanjong are both ok with me. Pick your favorite :)

Reply:
Chenjan seems to appear mostly across the web in stores and stuff, so I'd go with that :D

Reply:
Pai Chenjan it is then :)